pyaedt/edb_core/net_class.py

```diff
@@ -0,0 +1,181 @@
+    @property
+    def _active_layout(self):
+        """ """
+        return self._pedb.active_layout
+
+    @property
+    def _layout(self):
+        """ """
+        return self._pedb.layout
+
+    @property
+    def _cell(self):
+        """ """
+        return self._pedb.cell
+
+    @property
+    def db(self):
+        """Db object."""
+        return self._pedb.active_db
+
+    @property
+    def _logger(self):
+        """Edb logger."""
+        return self._pedb.logger
+
+
+class EdbExtendedNets(EdbCommon, object):
+    """Manages EDB methods for nets management accessible from `Edb.extended_nets` property.
+
+    Examples
+    --------
+    >>> from pyaedt import Edb
+    >>> edbapp = Edb("myaedbfolder", edbversion="2021.2")
+    >>> edb_nets = edbapp.extended_nets
+    """
+
+    @pyaedt_function_handler()
+    def __getitem__(self, name):
+        """Get  a net from the Edb project.
+
+        Parameters
+        ----------
+        name : str, int
+
+        Returns
+        -------
+        :class:` :class:`pyaedt.edb_core.edb_data.nets_data.EDBExtendedNetsData`
+
+        """
+        if name in self.extended_nets:
+            return self.extended_nets[name]
+        self._pedb.logger.error("Component or definition not found.")
+        return
+
+    def __init__(self, p_edb):
+        super().__init__(p_edb)
+        self._extended_nets = {}
+
+    @property
+    def extended_nets(self):
+        """Extended nets.
+
+        Returns
+        -------
+        dict[str, :class:`pyaedt.edb_core.edb_data.nets_data.EDBExtendedNetsData`]
+            Dictionary of extended nets.
+        """
+        for extended_net in self._layout.extended_nets:
+            self._extended_nets[extended_net.GetName()] = EDBExtendedNetData(self._pedb, extended_net)
+        return self._extended_nets
+
+    @pyaedt_function_handler
+    def create(self, name, net: list[str]) -> EDBExtendedNetData:
+        """
+
+        Parameters
+        ----------
+        name
+        nets
+
+        Returns
+        -------
+
+        """
+        if name in self._extended_nets:
+            self._pedb.logger.error("{} already exists.".format(name))
+            return False
+
+        extended_net = EDBExtendedNetData(self._pedb)
+        api_extended_net = extended_net.api_create(name)
+        if isinstance(net, str):
+            api_extended_net.add_net(net)
+        else:
+            api_extended_net.add_nets(net)
+
+        return self.extended_nets[name]
+
+
+class EdbDifferentialPair(EdbCommon, object):
+    """Manages EDB methods for nets management accessible from `Edb.differential_pairs` property.
+
+    Examples
+    --------
+    >>> from pyaedt import Edb
+    >>> edbapp = Edb("myaedbfolder", edbversion="2021.2")
+    >>> edb_nets = edbapp.differential_pairs
+    """
+
+    @pyaedt_function_handler()
+    def __getitem__(self, name):
+        """Get  a net from the Edb project.
+
+        Parameters
+        ----------
+        name : str, int
```

Comment:
```suggestion
        name : str, int
            Name or ID of the net.
```
Assumes that an integer data type would be an ID.